app/frontend/helpers/cliParser/types.ts

```diff
@@ -20,15 +22,23 @@ export const enum TxCertificateKeys {
   STAKEPOOL_REGISTRATION = 3,
 }
 
+export type Lovelace = BigInt
```

Comment:
If we are going with native BigInt, we should add a check for this in our initial "feature check" before the page gets loaded: https://github.com/vacuumlabs/adalite/blob/d7c1656937f36bc3331d06cdc4f1efae8264eecb/app/public/js/init.js#L2 as browsers such as Safari adopted it only recenly: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt - for this advanced feature it's currently not a big deal, so this is rather a note for the future when we update the wallet logic to handle big ints.

Probably worth considering though if we wouldn't be better off just using some library to not leave out users of older/non-mainstream browsers, but maybe it's too low of a percentage to bother as the majority of Safari users appears to have the 14 version already: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt